content/docs/tasks/security/authz-tcp/index.md

```diff
@@ -6,24 +6,23 @@ keywords: [security,access-control,rbac,tcp,authorization]
 ---
 
 This task covers the activities you might need to perform to set up Istio authorization, also known
-as Istio Role Based Access Control (RBAC), for TCP services in an Istio mesh. You can read more in
-[authorization](/docs/concepts/security/#authorization) and get started with a basic tutorial in
-Istio Security Basics.
+as Istio Role Based Access Control (RBAC), for TCP services in an Istio mesh. You can learn more about
+the Istio authorization in the [authorization concept page](/docs/concepts/security/#authorization).
 
 ## Before you begin
 
 The activities in this task assume that you:
 
-* Read [authorization](/docs/concepts/security/#authorization) concepts.
+* Read the [authorization concept](/docs/concepts/security/#authorization).
 
 * Follow the instructions in the [quick start](/docs/setup/kubernetes/quick-start/) to install Istio on
   Kubernetes **with authentication enabled**.
 
 * Enable mutual TLS (mTLS) authentication when running the [installation steps](/docs/setup/kubernetes/quick-start/#installation-steps).
 
 The commands used in this task assume the Bookinfo example application is deployed in the default
-namespace. To specify a namespace other than the default namespace, use `kubectl -n [my-namespace] ...`
-to specify the namespace.
+namespace. To specify a namespace other than the default namespace, use option `-n` to switch to the
+actual namespace in each commands.
```

Comment:
The words "switch" and "in each commands" can cause confusion particularly since the sentence start with "To specify...". I suggest:
..., use the `-n` option in the command.